fisher.lua

```diff
@@ -455,8 +477,14 @@ function check_incoming_chunk(id, original, modified, injected, blocked)
                 error_retry = true
             elseif new_status == 0 and old_status ~= 0 then
                 message(3, 'status changed to idle')
-                message(2, 'casting in %d seconds':format(settings.delay.cast))
-                windower.send_command('wait %d; lua i fisher cast':format(settings.delay.cast))
```

Comment:
Short circuit logic could be used here again instead of needed the if/else block.